tensorflow/python/ops/nn_ops.py

```diff
@@ -794,6 +794,71 @@ def _BatchNormGradShape(op):
 ops.RegisterShape("MaxPool")(common_shapes.max_pool_shape)
 
 
+@ops.RegisterShape("FusedResizeAndPadConv2D")
```

Comment:
note that this shape fn is not used since there is no python code for creating the op in the graph.  But the C++ version of the shape function would eventually be used server side.